test/bids_examples_eeg_rest_fmri.jsonld

```diff
@@ -1845,16 +1822,16 @@
       ],
       "experimentalApproach": [
         {
-          "@id": "https://openminds.ebrains.eu/instances/experimentalApproach/anatomy"
+          "@id": "https://openminds.ebrains.eu/instances/experimentalApproach/neuroimaging"
```

Comment:
this re-ordering seems fragile. I would expect this to break again in future. I suggest introducing sorting by @id

Reply:
Agreed, but should I add it to the openminds-python, or just sort it here?
As this was created by save method of the collection class in openminds-python, and then I have check it manually.

Reply:
yes, this should be done in openminds-Python